asdf_astropy/converters/unit/tests/test_masked.py

```diff
@@ -0,0 +1,41 @@
+import importlib.resources
+
+import asdf
+import numpy as np
+import pytest
+from astropy import units as u
+from astropy.utils.masked import Masked
+
+from asdf_astropy.converters.utils import MaskedClassesRequireAstropy71Error
+from asdf_astropy.testing.helpers import only_if_astropy_lt_7_1, skip_if_astropy_lt_7_1
+
+from . import resources
+
+MaskedQuantity = Masked(u.Quantity)
+
+
+@only_if_astropy_lt_7_1
```

Comment:
Let's move this outside this test file and add a module-level skip like mentioned in the other comment. That way we don't need to add the extra decorators.

Reply:
You mean, a new file, like `test_masked_not_implemented.py`?

Reply:
`test_quantity.py` seems fine to me something like:
```python
from asdf_astropy.tests.versions import ASTROPY_LT_7_1


@pytest.mark.skipif(not ASTROPY_LT_7_1, reason="MaskedQuantity support was added in astropy 7.1")
def test_masked_quantity_raises():
    import importlib.resources
    from asdf_astropy.converters.utils import MaskedClassesRequireAstropy71Error
    from . import resources

    def test_masked_quantity_raises():
        with pytest.raises(MaskedClassesRequireAstropy71Error):
            with asdf.open(importlib.resources.files(resources) / "test.asdf"):
                pass
```
pending adding `ASTROPY_LT_7_1` to that location (and also ignoring that this is using the test data file to be removed).

Reply:
Done.